docs/abi-spec.rst

```diff
@@ -313,7 +313,10 @@ A function description is a JSON object with the fields:
 - ``outputs``: an array of objects similar to ``inputs``, can be omitted if function doesn't return anything;
 - ``payable``: ``true`` if function accepts ether, defaults to ``false``;
 - ``stateMutability``: a string with one of the following values: ``pure`` (:ref:`specified to not read blockchain state <pure-functions>`), ``view`` (:ref:`specified to not modify the blockchain state <view-functions>`), ``nonpayable`` and ``payable`` (same as ``payable`` above).
-- ``constant``: ``true`` if function is either ``pure`` or ``view``
+
+.. warning::
+    Prior to version 0.5.0 there was an additional field ``constant``. In 0.5.0 this field has been removed.
+    Instead you can check whether stateMutability is ``pure`` or ``view``.
```

Comment:
Perhaps explain that `view` is the former `constant`?

Reply:
I'm not sure - for the abi it holds specifically that ``constant`` = (``pure`` or ``view``), so if people miss the ``constant`` field, that's what they should use instead - the fact that ``view`` replaces ``constant`` should rather be in the docs for state mutability itself and I think it's there already - I'll check.